templates/pages/target-platforms/boards/index.md

```diff
@@ -0,0 +1,19 @@
+----
+title: Boards
+description: Boards page
+page_type: summary_refine
+summary_pages:
+    - /core/get-started/raspberry-pi-2-3
+    - /core/get-started/compute-module-3
+    - /core/get-started/artik-5-10
+    - /core/get-started/dragonboard-410c
+    - /core/get-started/intel-nuc
+    - /core/get-started/intel-joule
+    - /core/get-started/kvm
+----
+
+# Boards
+
+All these boards are supported with official Ubuntu images, but they didn't necessarily went through the Ubuntu Certification program. You can find Ubuntu Certified hardware at [certification.ubuntu.com](https://certification.ubuntu.com).
```

Comment:
I would say:"
All of these boards are supported with official Ubuntu images, but they may not have completed the Ubuntu Certification program. You can find Ubuntu Certified hardware at [certification.ubuntu.com (https://certification.ubuntu.com).

Reply:
+1